react_app/src/components/Misc/PlayerDetails.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+
+import Button from "@material-ui/core/Button";
+
+class PlayerDetails extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getUserName() {
+    return this.props.userName;
+  }
+  getTeamName() {
+    if (this.props.teamName) {
+      return "[" + this.props.teamName + "]";
+    }
+    return "Solo";
+  }
+
+  render() {
+    if (!this.props.isAuthenticated) {
```

Comment:
Why check auth down here in a child component? If you are checking it at the root component your app will just show the login page right?

Reply:
I could lift this up to the parent, but essentially if you're not logged in it won't display the currently logged in user their team information